qt/widgets/mplcpp/src/Colors.cpp

```diff
@@ -176,8 +179,11 @@ Python::Object SymLogNorm::tickLocator() const {
   // Create log transform with base=10
   auto transform = scaleModule().attr("SymmetricalLogTransform")(
       10, Python::Object(pyobj().attr("linthresh")), m_linscale);
+  std::vector<float> subsVector(10);
```

Comment:
Could we add a small comment describing what this and the next line is doing? 
Particularly when using hard-coded numbers it's good to explain where they come from. It's also not quite clear that it's anything to do with the minor ticks until reading the matplotlib documentation.